habitat_baselines/rl/hrl/skills/nn_skill.py

```diff
@@ -113,34 +115,31 @@ def _internal_act(
         return full_action, rnn_hidden_states
 
     @classmethod
-    def from_config(cls, config, observation_space, action_space, batch_size):
+    def from_config(
+        cls, config, observation_space, action_space, batch_size, full_config
```

Comment:
Same here regarding full config. What part of it is used and what is the difference between config and full_config.